src/python/pants/backend/project_info/dependees.py

```diff
@@ -27,16 +27,18 @@ class AddressToDependees:
 @rule
 async def map_addresses_to_dependees() -> AddressToDependees:
     # Get every target in the project so that we can iterate over them to find their dependencies.
-    all_explicit_targets = await Get(Targets, AddressSpecs([DescendantAddresses("")]))
+    all_expanded_targets, all_explicit_targets = await MultiGet(
+        Get(Targets, AddressSpecs([DescendantAddresses("")])),
+        Get(UnexpandedTargets, AddressSpecs([DescendantAddresses("")])),
+    )
+    all_targets = {*all_expanded_targets, *all_explicit_targets}
```

Comment:
Consider `FrozenOrderedSet` for determinism. I don't think it makes a difference the order of `Get`s in a `MultiGet`, though.